corehq/apps/reports/filters/users.py

```diff
@@ -461,14 +461,31 @@ def for_reporting_group(cls, group_id):
         }
 
 
+class AffectedUserFilter(ExpandedMobileWorkerFilter):
+    label = _("Affected User")
+
+
 class ChangedByUserFilter(ExpandedMobileWorkerFilter):
     slug = "changed_by_user"
-    label = ugettext_lazy("Changed By User(s)")
+    label = ugettext_lazy("Modified by User(s)")
 
     def get_default_selections(self):
         return [('t__6', _("[Web Users]"))]
 
 
+class PropertyFilter(BaseSingleOptionFilter):
+    label = ugettext_noop('Modified Property')
+    default_text = ugettext_noop('Select Property')
+    slug = 'property'
```

Comment:
nit: can this be `user_property`? just to be more explicit. Even though the filter is for users but that is a bit more implicit

Reply:
sure - c18483cf2df